fortex/health/processors/scispacy_processor.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+SciSpacy Processor
+"""
+from typing import Dict, Set
+import importlib
+
+import spacy
+from forte.common import Resources
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+
+# pylint: disable=unused-import
```

Comment:
don't disable this, remove the unused-import

Reply:
I found out that, James actually used this because "HyponymDetector" and "AbbreviationDetector" because it is used by pipes in test otherwise it will give same error as earlier. 

```
config = {
            "entry_type": "ft.onto.base_ontology.Document",
            "model_name": "en_core_sci_sm",
            "pipe_name": "abbreviation_detector",
        }
```